api/src/routes/email/router.ts

```diff
@@ -0,0 +1,7 @@
+import { Router } from "express";
+import * as emailHandlers from "../../handlers/email";
+
+export const emailRouter = Router();
+
+emailRouter.post("/affirmation", emailHandlers.affirmationPost);
```

Comment:
I think the output will be configured to something like `api:9000/forms` At most `api:9000/forms/affirmation` but I doesn't make sense to expose it as `email/affirmation`, since it won't always be an email. 

in a form json, you can add metadata, and that will be sent with question data.
```
metadata: {
  type: "affirmationOrAffidavit" // "certificateOfNoImpediment 
}
```

then when the payload comes in, you can check metadata.type on how you want to process the submission, but generally, all submissions would be handled like a form.